packages/salesforcedx-vscode-core/src/commands/forceSourceDeploy.ts

```diff
@@ -13,7 +13,10 @@ import {
   Row,
```

Comment:
All the changes done lately around deploy and push have left the name of this file a bit obsolete, it doesn't reflect the code's purpose anymore. What do you think about renaming it ?

Reply:
cc @allileong

Reply:
Do you think it should be something more like "baseDeployCommand"?

Reply:
That sounds good to me 👍